diag_table_source.yaml

```
@@ -0,0 +1,330 @@
+            tauuo: # surface_downward_x_stress taux
+            tauvo: # surface_downward_y_stress tauy
+            uhbt: # Barotropic zonal transport averaged over a baroclinic step
+            vhbt: # Barotropic meridional transport averaged over a baroclinic step
+            ustar: # Surface friction velocity = [(gustiness + tau_magnitude)/rho0]^(1/2)
+            umo_2d: # Ocean Mass X Transport Vertical Sum
+            vmo_2d: # Ocean Mass Y Transport Vertical Sum
+            pbo: # Sea Water Pressure at Sea Floor
+
+    'monthly 2d max fields':
+        defaults:  # these can be overridden for individual fields below
+            file_name_dimension: 2d  # descriptor for filename, e.g. 3d, 2d, scalar
+            reduction_method: max  # mean, snap, rms, pow##, min, max, or diurnal##
+        fields:
+            mlotst: # Mixed layer depth (delta rho = 0.03) MLD_003
+
+    'monthly 2d min fields':
+        defaults:  # these can be overridden for individual fields below
+            file_name_dimension: 2d  # descriptor for filename, e.g. 3d, 2d, scalar
+            reduction_method: min  # mean, snap, rms, pow##, min, max, or diurnal##
+        fields:
+            tos: # Sea Surface Temperature
+
+    'daily scalar timeseries':
+        defaults:  # these can be overridden for individual fields below
+            file_name_dimension: scalar  # descriptor for filename, e.g. 3d, 2d, scalar
+            file_name:  # String, or list of components, for file name (without trailing ".nc").
+            # If a list, its elements are a mixture of strings and dictionaries.
+            # String list elements are concatenated, each preceded by file_name_separator.
+            # Dictionary list elements must have one key-value pair; the value is a list
+            # containing 1 or more strings to be concatenated, each preceded by the key
+            # (a string) instead of file_name_separator. The key may be an empty string,
+            # which is useful if file_name_date begins with %, since FMS prepends _ .
+            # All filename list string elements (other than field_name) must exist as keys in global_defaults.
+            # If file_name_date is used, it must be the last item.
+            # Key values are looked up, converted to strings and then substituted using file_name_substitutions.
+            # Empty strings (and their preceding separator) are ignored if file_name_omit_empty is true.
+            # A separator is not used prior to the first item.
+                - file_name_prefix
+                - file_name_dimension
+                - output_freq
+                - '':
+                    - output_freq_units
+                - file_name_date
+            output_freq_units: days  # time units for output: years, months, days, hours, minutes, or seconds
+            reduction_method: mean  # mean, snap, rms, pow##, min, max, or diurnal##
```

```suggestion
            reduction_method: snap  # mean, snap, rms, pow##, min, max, or diurnal##
```
snapshots of global integrals will probably give better parallel performance than averages, as they don't require collectives and synchronisation of all MPI ranks every timestep